client/src/components/Button.tsx

```diff
@@ -45,52 +46,73 @@ interface StyledButtonProps {
   buttonStyle: ButtonStyle;
 }
 
+const disabledButton = css`
+  opacity: 0.4;
+  /* stylelint-disable-next-line plugin/no-unsupported-browser-features -- Seems to actually work on Safari */
+  cursor: not-allowed;
+`;
+
 const StyledButton = styled.button<StyledButtonProps>`
-  border-radius: 5px;
+  border-radius: 6px;
   cursor: pointer;
-  margin: 10px 10px 10px 0;
-  padding: 6px 20px;
+  box-shadow: rgba(0, 0, 0, 0.18) 0 3px 5px;
```

Comment:
Pitäskö `box-shadow` värin olla `theme.ts` tiedostossa?

Reply:
Toisaalla oli arvona `rgba(0, 0, 0, 0.15) 0 3px 5px`. Voiko noita eri `box-shadow` tyylejä jotenkin koostaa `themes.ts` tiedostoon vai onko ne liian käyttötapauskohtaisia?

Reply:
Luulen, että noi varjot on lähinnä sattumalta samat ja oikeastaan esim. FavoriteButtonissa ei ehkä ees pitäis olla varjoa (mutta siinä on nyt, koska se on ainakin yhdellä sivulla muuten sekoitettavissa ei-klikattaviin ikoneihin) eli ei ehkä ainakaan vielä kannata alkaa erityisesti yhtenäistää.